[LLM] New docs (#2828)
scaleway/docs-content: ai-data/llm-inference/how-to/monitor-deployment.mdx

---
meta:

>   title: How to monitor a LLM Inference deployment

Reviewer: ```suggestion
  title: How to monitor an LLM Inference deployment
```

>   description: This page explains how to monitore a LLM Inference deployment

Reviewer: ```suggestion
  description: This page explains how to monitor an LLM Inference deployment
```

content:

>   h1: How to monitor a LLM Inference deployment

Reviewer: ```suggestion
  h1: How to monitor an LLM Inference deployment
```

>   paragraph: This page explains how to monitore a LLM Inference deployment

Reviewer: ```suggestion
  paragraph: This page explains how to monitor an LLM Inference deployment
```

tags: ai monitoring
dates:
  validation: 2024-03-06
  posted: 2024-03-06
categories:
  - ai-data
---

> Monitoring allows you to monitor your deployment.

Reviewer: ```suggestion
This documentation page shows you how to monitor your LLM Inference deployment with [Cockpit](/observability/cockpit/quickstart/).
```

<Macro id="requirements" />

  - A Scaleway account logged into the [console](https://console.scaleway.com)
  - An [LLM Inference deployment](/ai-data/llm-inference/quickstart/)

>   - An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)

Reviewer: Cockpit activation is going to be deprecated on the 2nd of April. Cockpits will be automatically activated. Should we remove this requirement?

## How to monitor your LLM dashboard

1. Click **LLM Inference** in the **AI & Data** section of the [Scaleway console](https://console.scaleway.com) side menu. A list of your deployments displays.
2. Click a deployment name or <Icon name="more" /> > **More info** to access the deployment dashboard.
3. Click the **Monitoring** tab of your deployment. The Cockpit overview displays.
4. Click **Open Grafana metrics dashboard** to open your Cockpit's Grafana interface.

> 5. Authenticate with your Grafana credentials. The Grafana dashboard displays.

Reviewer: ```suggestion
5. Authenticate with your [Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/). The Grafana dashboard displays.
```
